es717x/src/main/scala/tech/beshu/ror/es/ReadonlyRestEsConfig.scala

```diff
@@ -21,20 +21,18 @@ import monix.eval.Task
 import tech.beshu.ror.configuration.{EnvironmentConfig, FipsConfiguration, MalformedSettings, RorBootConfiguration, RorSsl}
 import tech.beshu.ror.utils.ScalaOps._
 
-import java.nio.file.Path
-
 final case class ReadonlyRestEsConfig(bootConfig: RorBootConfiguration,
```

Comment:
Let's move this class to the core module (I don't see any differences between current modules)

Reply:
good catch 👍